packages/model-viewer/src/features/loading.ts

```diff
@@ -310,6 +311,8 @@ export const LoadingMixin = <T extends Constructor<ModelViewerElementBase>>(
       const ModelViewerElement: ModelViewerGlobalConfig =
           (self as any).ModelViewerElement || {};
 
+      Renderer.powerPreference = ModelViewerElement.powerPreference || DEFAULT_POWER_PREFERENCE;
```

Comment:
I think you still need to define the `powerPreference` property somewhere, and probably better in `model-viewer-base` than `loading`.

Reply:
It seems to be working when I check in debugger for the case I introduced in examples.

Reply:
How did you check?

Reply:
I would recommend checking this: https://developer.mozilla.org/en-US/docs/Web/API/WebGLRenderingContext/getContextAttributes